formatTest/unit_tests/expected_output/features403.re

```diff
@@ -13,7 +13,11 @@ type nonrec u 'a =
 type expr 'a =
   | Val {value: 'a} :expr 'a
   | Add
-      {left: expr int, right: expr int} :expr int
+      {
+        left: expr int,
+        right: expr int
+      }
+      :expr int
```

Comment:
This may be worth lifting to the previous line too.  I'm less certain here though.